### PR TITLE
Decay learning rate

### PR DESCRIPTION
**Pull request by normanheckscher**
New decay learning rate based on the following code:
https://github.com/sherjilozair/char-rnn-tensorflow/pull/70

### CODE REVIEW

**Comment by hunkim** [comment]
Decay learning rate improves learning?
